lib/experiment_2_slugs

```diff
@@ -0,0 +1,4880 @@
+guidance-and-regulation-with-1-topic-ce3f6f1b86
```

Comment:
I'm a bit surprised about this file being in lib, it feels more like config? perhaps useful to give it a .txt extension too.

Reply:
Sure, I've done that. I've got a few checklist items to remove it soonish anyway.